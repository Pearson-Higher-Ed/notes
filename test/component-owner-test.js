import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from 'react-intl';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import { InternationalSupport } from '@pearson-incubator/aquila-js-core';
import ComponentOwner from '../src/js/component-owner';
import msgObject from '../translations';

const intlObj = new InternationalSupport(msgObject, 'en');

/* eslint-disable */

it('renders the component correctly', () => {
 const tree = renderer
   .create(<MuiThemeProvider><IntlProvider 
     locale={intlObj.getLocale()} 
     messages={intlObj.getMessages()}>
     <ComponentOwner clickNoteHandler={function(){}}
          removeNoteHandler={function(){}} ></ComponentOwner>
     </IntlProvider></MuiThemeProvider>)
   .toJSON();
 expect(tree).toMatchSnapshot();
});