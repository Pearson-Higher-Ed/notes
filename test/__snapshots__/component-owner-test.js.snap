// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component correctly 1`] = `
<div
  id="notes"
  role="main"
>
  <div
    className="notes-body"
  />
  <div
    aria-live="assertive"
    className="reader-only"
    id="notes-assert-container"
    role="alert"
  />
</div>
`;
